pkg/build/strategies/onbuild/onbuild_test.go

```diff
@@ -48,15 +48,15 @@ func checkDockerfile(fs *test.FakeFileSystem, t *testing.T) {
 		t.Errorf("%v", fs.WriteFileError)
 	}
 	if fs.WriteFileName != "upload/src/Dockerfile" {
-		t.Errorf("Expected Dockerfile in 'upload/src/Dockerfile', got %v", fs.WriteFileName)
+		t.Errorf("Expected Dockerfile in 'upload/src/Dockerfile', got %q", fs.WriteFileName)
 	}
 	if !strings.Contains(fs.WriteFileContent, `ENTRYPOINT ["./run"]`) {
-		t.Errorf("The Dockerfile does not set correct entrypoint:\n %s\n", fs.WriteFileContent)
```

Comment:
No as is, but updated to `:\n%s`, removing unnecessary spaces and trailing new line.